tests/test_fips.py

```diff
@@ -152,3 +186,53 @@ def fips_mode_setup_check(container_per_test: ContainerData) -> None:
 )
 def test_openssl_fips_hashes(container_per_test: ContainerData):
     openssl_fips_hashes_test_fnct(container_per_test)
+
+
+@pytest.mark.parametrize(
+    "container_per_test", FIPS_GNUTLS_TESTER_IMAGES, indirect=True
+)
+@pytest.mark.skipif(
```

Comment:
We don't want to skip this unconditionally, the image should be fixed eventually:

```suggestion
@pytest.mark.xfail(
```